app/frontend/src/assets/scss/style.scss

```diff
@@ -1,3 +1,12 @@
+/* Typography */
+body .v-application {
+  font-family: -apple-system, BlinkMacSystemFont, 'BCSans', ‘Noto Sans’, Verdana, Arial, sans-serif !important;
```

Comment:
The quotes around Noto Sans looks a bit strange - is this intended?

Reply:
well spotted! i think the official way is double quotes around names made of more than one word. i'll change this